src/blocks/date-block/edit.js

```diff
@@ -0,0 +1,55 @@
+/**
```

Comment:
Rename to `event-date` take a look at `gatherpress/core/classes/class-block.php` too. I have set up helpers for getting this block working.